examples/rustapi_module/tests/test_datetime.py

```diff
@@ -80,7 +83,8 @@ def test_invalid_date_fails():
         rdt.make_date(2017, 2, 30)
 
 
-@given(d=dates())
+# Feeding this tests dates from too early will cause `get_timestamp` to raise.
```

Comment:
What does it mean?

Reply:
I was getting crashes in the python test when hypothesis was feeding really early dates which are completely unrelated to the rust end of things. It might be a bug in PyPy's interpreter, since In cpython it just returns a large negative number.

I guess it's because trying to make dates from before the epoch (from timestamp) doesn't make a lot of sense.

![image](https://user-images.githubusercontent.com/2467993/54739853-2f406a00-4bc2-11e9-9e94-a48b5457c294.png)

Reply:
I see, thanks.